internal/util/testqueue/passthroughqueue.go

```diff
@@ -38,6 +46,7 @@ func (q *PassthroughQueue) GetQueue() <-chan *message.Message {
 
 // Pass passes a message through the queue
 func (q *PassthroughQueue) Pass(msg *message.Message) error {
+	q.t.Logf("Passing message through queue: %s", msg.UUID)
```

Comment:
Would it make sense to log the event topic too?

Reply:
That's not available through the message structure. Maybe if we start having several test queues in our tests, but that's not the case right now.